PluginsAndFeatures/azure-toolkit-for-intellij/.gitignore

```diff
@@ -44,3 +44,6 @@ gen-external-apklibs/
 
 # Jar dependencies generated in build
 **/resources/spark/spark-tools-*.jar
+
+# IntelliJ
+.idea/
```

Comment:
missing carriage return (similar to josh's comment below)

Reply:
added line break at the end